programs/eosioc/main.cpp

```diff
@@ -436,6 +436,25 @@ struct set_action_permission_subcommand {
    }
 };
 
+struct currency_stats {
+   asset supply;
+};
+
+FC_REFLECT(currency_stats, (supply));
+
+static currency_stats parse_currency_stats( fc::variant const& row ) {
```

Comment:
`const fc::variant& row` is style used elsewhere. Also in `parse_currency_balance`.